ingestion/functions/parsing/germany/germany.py

```diff
@@ -0,0 +1,105 @@
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+if "lambda" not in sys.argv[0]:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))), "common/python"
+        )
+    )
+import parsing_lib
+
+_DATE_INDEX = 8
+_AGE_INDEX = 4
+_GENDER_INDEX = 5
+_CASECOUNT = 6
+_DEATHCOUNT = 7
+_RECOVERYCOUNT = 15
+_ADMIN1 = 2
+_ADMIN2 = 3
+
+
+def convert_date(raw_date):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+    """
+    date = datetime.strptime(raw_date, "%Y/%m/%d %H:%M:%S")
+    return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender):
+    if raw_gender == "M":
+        return "Male"
+    if raw_gender == "W":
+        return "Female"
+
+
+def convert_demographics(gender: str, age: str):
+    demo = {}
+    if gender:
+        demo["gender"] = convert_gender(gender)
+    if age:
+        age_range = age.split("-")
+        if len(age_range) > 1:
+            demo["ageRange"] = {
+                "start": float(age_range[0][1:3]),
```

Comment:
Perhaps toss this into a separate function with a brief explanation (i.e. why the data format necessites `1:3`)?